client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -0,0 +1,36 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { withInstanceId } from '@wordpress/compose';
+
+class Edit extends Component {
+	handleChange = event => {
+		const paymentId = parseInt( event.target.value, 10 );
+		if ( paymentId ) {
+			this.props.setAttributes( { paymentId } );
+		} else {
+			this.props.setAttributes( { paymentId: undefined } );
+		}
+	};
+
+	render() {
+		const { attributes, instanceId } = this.props;
+		const inputId = `payment-id-${ instanceId }`;
+		return (
+			<Fragment>
+				<label htmlFor={ inputId }>Simple Payment ID</label>
+				<input
```

Comment:
Throws:
> Warning: A component is changing an uncontrolled input of type number to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component.

Reply:
I've just pushed a fix for this.

This is just getting something into place, warnings shouldn't be too concerning at this point.